docs/components/Popover/Popover.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Overlays/Popover/Docs" />
+
+# Popover
+
+A Popover displays floating informative and actionable content positioned in
+relation to a target. The Popover can contain content, media, or other
+components.
+
+## Usage guidelines
+
+Some scenarios for Popover include the following:
+
+### Related components
+
+To add a menu button that presents multiple actions to the user, use
+[Menu](../?path=/docs/components-navigation-menu--menu).
+
+To add a hint about a UI element's function in a permanent fashion (ie revealing
+a Button's label on hover), use
+[Tooltip](../?path=/docs/components-overlays-tooltip--tooltip).
+
+To add an inline informational element that the user can dismiss, consider if
+[Banner](../?path=/docs/components-banner--banner) is the right fit for your use
```

Comment:
The banner link is broken. I know it might change, but currently it is `path=/docs/components-status-and-feedback-banner--banner`